BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -97,14 +97,25 @@ - (instancetype)initWithURL:(NSURL *)URL HTTPMethod:(BOXAPIHTTPMethod *)HTTPMeth
     self = [super init];
     if (self != nil)
     {
+        
+        // If we are not a POST method, postParams will be nil and we will not add a body.
+        // If we are a POST method, all api calls will append our token to the body prior to encoding.
+        NSMutableDictionary *POSTParams = [body mutableCopy];
+        if ([session isKindOfClass:[BOXOAuth2Session class]]) {
+            BOXOAuth2Session *currentSession = (BOXOAuth2Session *)session;
+            if (currentSession.additionalTokenGrantParameters.count > 0) {
```

Comment:
This may be a good time to rename the session property from additionalTokenGrantParameters?
It's relating to auth/tokengranting/etc, but it's a bit more generic in that it no longer is used just in the token resource API calls.

Maybe just additionalGrantParameters? What do you think would be appropriate?